src/zenml/cli/stack_components.py

```diff
@@ -195,6 +195,13 @@ def generate_stack_component_register_command(
         required=True,
         type=str,
     )
+    @click.option(
+        "--metadata",
+        "-m",
+        "metadata",
+        help="Metadata to be associated with the component.",
+        multiple=True,
+    )
```

Comment:
Is there any place where the expectations for what to input here are communicated to the user? Maybe you should add this to the xamples in src/zenml/cli/__init__.py so it is added to the api_docs. Also maybe the help messages should contain an example.

Reply:
Good idea. I have added the expectation to both the `__init__` file and the `help` message.